diff --git a/node_modules/@coderline/alphatab/dist/alphaTab.js b/node_modules/@coderline/alphatab/dist/alphaTab.js
index c63d1e6..1c2ad45 100644
--- a/node_modules/@coderline/alphatab/dist/alphaTab.js
+++ b/node_modules/@coderline/alphatab/dist/alphaTab.js
@@ -3645,6 +3645,10 @@
             let index = this.notes.indexOf(note);
             if (index >= 0) {
                 this.notes.splice(index, 1);
+                // https://github.com/CoderLine/alphaTab/pull/933
+                if (note.isStringed) {
+                    this.noteStringLookup.delete(note.string);
+                }
             }
         }
         getAutomation(type) {
